java-bookstore/src/main/java/com/lambdaschool/bookstore/controller/BooksController.java

```diff
@@ -0,0 +1,46 @@
+package com.lambdaschool.bookstore.controller;
+
+import com.lambdaschool.bookstore.model.Book;
+import com.lambdaschool.bookstore.model.ErrorDetail;
+import com.lambdaschool.bookstore.service.BookService;
+import io.swagger.annotations.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/books")
+public class BooksController
+{
+	@Autowired
+	private BookService bookService;
+
+	@ApiOperation(value = "Returns all books, pageable")
```

Comment:
Documentation looking good! You could add a description for each method, but it's looking good as is!